_ingest-pipelines/processors/json.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: default
+title: JSON
+parent: Ingest processors
+nav_order: 170
+---
+
+# JSON processor
+
+The `json` processor serializes a string-valued field into a map of maps, which can be useful for various data processing and enrichment tasks.
+
+The following is the syntax for the `json` processor:
+
+```json
+{
+  "processor": {
+    "json": {
+      "field": "<field_name>",
+      "target_field": "<target_field_name>",
+      "add_to_root": <boolean>
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+## Configuration parameters
+
+The following table lists the required and optional parameters for the `json` processor.
+
+Parameter | Required/Optional | Description |
+|-----------|-----------|-----------|
+`field` | Required | The name of the field containing the JSON-formatted string to be deserialized.
+`target_field` | Optional | The name of the field where the deserialized JSON data will be stored. If `target_field` is unspecified, the value is stored in field. If `target_field` exists, it is overwritten.
```

Comment:
The last two sentences here have me a bit confused. Can we check for clarity/accuracy?